lmdeploy/turbomind/deploy/converter.py

```diff
@@ -226,8 +191,6 @@ def main(model_name: str,
         quant_path (str): Path of the quantized model, which can be None.
         group_size (int): a parameter used in AWQ to quantize fp16 weights
             to 4 bits
-        trust_remote_code (bool):  Whether or not to allow for custom models
-            defined on the Hub in their own modeling files. Defaults to False
```

Comment:
we are using this argument. In my test, I have to add `--trust-remote-code` for `lmdeploy convert` command during converting local models.

Reply:
@irexyc used to suggest removing it
All right, I can remove this argument

Reply:
Simply set `trust-remote-code` to true by default.